Sensorable/wear/src/main/java/com/sensorable/WearSensorDataSender.java

```diff
@@ -43,14 +43,19 @@ public void run() {
                 try {
                     capabilityInfo = Tasks.await(
                             Wearable.getCapabilityClient(context).getCapability(
-                                    WEAR_DATA_RECEPTION, CapabilityClient.FILTER_REACHABLE));
+                                    WEAR_DATA_RECEPTION, CapabilityClient.FILTER_REACHABLE)
+                    );

```

Comment:
Improve the format, maybe something like: 
capabilityInfo = Tasks.await(
                            Wearable
                            .getCapabilityClient(context)
                            .getCapability(WEAR_DATA_RECEPTION, CapabilityClient.FILTER_REACHABLE)
);